### DISCUSSION TITLE
Implementation for ACE Descriptors tutorial

### DISCUSSION THREAD

**Discussion by ireaml** [JuliaMolSim/PoSH.jl#148] [Q&A]
Hi!
Was interested in using `ACE` to generate a structural descriptor, as exemplified in the [ACE Descriptors tutorial](https://acesuit.github.io/ACE1pack.jl/dev/literate_tutorials/descriptor/#ACE-Descriptors), but seems like it's broken. It mentions that an update to the pair potential implementation is required, but not sure which implementation this corresponds to?

Thanks very much!

**Comment by cortner** [comment]
Hi Irea - sorry for the flow response, somehow this fell to the bottom of my emails. Please start by taking a look at ACE1pack.jl rather than ACE.jl. This is where you will be able to find everything.

**Reply by cortner** [reply]
https://github.com/ACEsuit/ACE1pack.jl

**Reply by cortner** [reply]
I just opened the Discussion board at `ACE1pack.jl` - please re-open the thread there. If you describe in a little more detail what you need then I'll be very happy to help you find things. Depending on what you want it might already be user-friendly packaged and ready to go or we can see how to make it available.

**Reply by ireaml** [reply]
Thanks Christoph! I raised an issue in `ACE1pack.jl` (https://github.com/ACEsuit/ACE1pack.jl/issues/81)
